packages/netskope/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml

```
@@ -1324,6 +1324,22 @@ processors:
           List setList = new ArrayList(new HashSet(l));
           ctx.user.email = setList;
         }
+  - script:
+    if: ctx.user?.email != null
```

Suggest using `instanceof List` since the code assumes the value is a list.

```suggestion
    if: ctx.user?.email instanceof List
```

---

The previous processor already forces `user.email` to be a list, so I was just testing if it exists or not.

I changed to validate it with `instanceOf List`